src/test/integration/intercom/test_intercom_common.py

```diff
@@ -29,8 +28,7 @@ def test_small_file(listener):
     check_file(b'this is a test', listener)
 
 
-# ToDo: fix intercom for larger values
-@pytest.mark.skip(reason='fixme plz')
+@pytest.mark.skip(reason='should not run on CI')
```

Comment:
What is the option to run this?

Reply:
You can run it now with environment variable `RUN_EXPENSIVE_TESTS` set to 1 but make sure you have at least 4 GB of available RAM